packages/yarnpkg-builder/sources/commands/build/bundle.ts

```diff
@@ -43,6 +43,9 @@ export default class BuildBundleCommand extends Command {
   @Command.Boolean(`--no-minify`, {description: `Build a bundle for development, without optimizations (minifying, mangling, treeshaking)`})
   noMinify: boolean = false;
 
+  @Command.Boolean(`--include-source-map`, {description: `Includes a source map in the bundle`})
+  includeSourceMap: boolean = false;
```

Comment:
I'll just rename that into `--source-map`, which I think is more in line with the general option style.